Makefile

```diff
@@ -2,7 +2,8 @@
 MKFILE_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 CPP_DIR := $(MKFILE_DIR)timemachine/cpp/
 INSTALL_PREFIX := $(MKFILE_DIR)timemachine/
-PYTEST_CI_ARGS := --color=yes --cov=. --cov-report=html:coverage/ --cov-append --durations=100 --hypothesis-profile ci
+# Conditionally set pytest args, to be able to override in CI
+PYTEST_CI_ARGS ?= --color=yes --cov=. --cov-report=html:coverage/ --cov-append --durations=100 --hypothesis-profile ci
```

Comment:
Exactly, ad-hoc modifications. 

I used it to avoid having to run 4 hours of tests to test one test that was 3 hours in. As the test didn't fail on my machine

Reply:
It is not currently overridden, but if you kick off a gitlab pipeline you can set PYTEST_CI_ARGS as a environment variable